website/docs/concepts/nodes-networks.md

```diff
@@ -0,0 +1,111 @@
+---
+id: nodes-networks
+title: Nodes and networks
+sidebar_label: Nodes and networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ClientStackPng from '@site/static/img/client-stack.png';
+import NetworkPng from '@site/static/img/network.png';
+import NetworkLayersPng from '@site/static/img/network-layers.png';
+
+
+Ethereum is a decentralized **network** of **nodes** that communicate peer-to-peer:
+
+<img style={{width: 100 + '%', margin: 'auto', display: 'block', maxWidth: 461 + 'px'}} src={NetworkPng} /> 
+
+
+## Nodes
+
+An Ethereum **node** is a running instance of Ethereum client software. There are two primary types of nodes in Ethereum: **execution nodes** and **beacon nodes**. Colloquially, a "node" refers to an execution node and beacon node working together.
+
+When users stake 32 ETH to participate in Ethereum's proof-of-stake consensus mechanism, they use validator client software like Prysm to stake their ETH and run a **validator**. Validators are not nodes - a validator can be thought of as an extension of a beacon node that manages validator keys and duties. Validators depend on beacon nodes, which depend on execution nodes:
```

Comment:
```suggestion
When users stake 32 ETH to participate in Ethereum's proof-of-stake consensus mechanism, they use a separate piece of software called a **validator client**, which connects to their Prysm beacon node. This is special software that validator keys and duties such as producing new blocks and voting on others' blocks. A validator client depends on a beacon node, which depend on an execution node:
```

Reply:
Do we want to normalize referring to "validator clients" as "validators" for consistency?

Execution client -> Execution node (running instance)
Beacon node client -> Beacon node (running instance)
Validator client -> Validator (running instance)